src/argilla/client/feedback/dataset/remote/base.py

```diff
@@ -312,7 +320,7 @@ def pull(self) -> "FeedbackDataset":
     def push_to_argilla(
         self,
         name: Optional[str] = None,
-        workspace: Optional[Union[str, Workspace]] = None,
+        workspace: Optional[Union[str, "Workspace"]] = None,
         show_progress: Optional[bool] = None,
```

Comment:
Indeed we can make this `**kwargs`, as it won't use them, so there's no need to pre-define them